### PR TITLE
ci: Add docker to goreleaser

### PR DESCRIPTION
**Pull request by cabrinha**
Let's build svu into a Docker image. The way I've proposed it here, the image comes in around 3.5MB 🚀

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM alpine as build
+
+RUN apk update && \
+  apk add curl bash && \
+  curl -sfL https://install.goreleaser.com/github.com/caarlos0/svu.sh | bash -s -- -b /usr/local/bin
+
+FROM scratch
+
+COPY --from=build /usr/local/bin/svu /usr/local/bin/svu
```
Comment (caarlos0) [inline Dockerfile:9]: this can be just

```suggestion
FROM scratch

COPY svu /usr/local/bin/svu
```

Comment (cabrinha) [reply Dockerfile]: Updated. Should we still keep the `ENTRYPOINT` there?

**Comment by caarlos0** [comment]
Thanks @cabrinha !

**Comment by caarlos0** [comment]
refs #28

**Comment by cabrinha** [comment]
@caarlos0 thanks for getting this merged so quickly. When will we be able to pull the image?

```
❯ docker pull ghcr.io/caarlos0/svu
Using default tag: latest
Error response from daemon: Head https://ghcr.io/v2/caarlos0/svu/manifests/latest: denied
```

**Comment by cjtim** [comment]
Can you provided the docs for how to use `svu` from docker image? Thanks

**Comment by caarlos0** [comment]
> @caarlos0 thanks for getting this merged so quickly. When will we be able to pull the image?

sorry, it was private by default, just changed to public.

**Comment by caarlos0** [comment]
> Can you provided the docs for how to use `svu` from docker image? Thanks

added to readme

**Pull request merged**